gui-libs/greetd/greetd-0.6.1-r1.ebuild

```diff
@@ -103,7 +103,11 @@ src_install() {
 	insinto /etc/greetd
 	doins config.toml
 
-	systemd_dounit greetd.service
+	if use systemd; then
+		systemd_dounit greetd.service
+	else
+		newinitd "${FILESDIR}"/${PN}.init ${PN}
+	fi
```

Comment:
Hi,

You can simplify this by removing the `systemd` use flag and simply install both files.
Actually this is even prohibited, see https://wiki.gentoo.org/wiki/Project:Systemd/Ebuild_policy#General_guidelines

Reply:
Updated. Thanks for the pointer.